docs/develprocess.md

```diff
@@ -702,12 +710,11 @@ Read [the usual warnings about the installation of third party RPMs on XCP-ng.](
 
 To achieve this:
 * Change the RPMs in the `Packages/` directory. If you add new packages, be careful about dependencies, else they'll fail to install and the whole installation process will fail.
-* Copy the file called `repodata/{varying_checksum_here}-groups.xml to a temporary location such as `/tmp/groups.xml`
 * Modify it to add or remove RPMs from the groups. There are two groups and both will be installed, so it's not very important if you don't know which one to modify. Just pick one. You don't need to add all the dependencies: they will be pulled automatically if you made them available in `Packages/`.
```

Comment:
This isn't true anymore.

Let's replace it with "If you need to add new RPMs not just replace existing ones, they need to be pulled by another existing RPM as dependencies. If there's none suitable, you can add the dependency to the `xcp-ng-deps` RPM (link to the git repo)"

Reply:
Where?